### ISSUE TITLE
Affixation alignment issue

### ISSUE THREAD

**Issue by bdarcus** [bdarcus/bibtex-actions#247]
@aikrahguzar:

![image](https://user-images.githubusercontent.com/1134/130226558-e6ce7223-ac8b-4ba1-a207-136481a4b608.png)

The display of the "file" symbol is I assume a bug in my Emacs 28 build, but note the alignment here; it actually impacts the entire row, though I only show the beginning.

Can you check this when you get a chance please? I assume the issue is in this line, though am unclear why:

https://github.com/bdarcus/bibtex-actions/blob/5356d118fd646d5aa04254a31c523c3e642b412b/bibtex-actions.el#L374

**Comment by aikrahguzar** [comment]
This caused by the file symbol we have by default. Its width is 1 but it clearly occupies more than one column which is what is causing the issue. The really correct way to solve that would be to truncate by pixels not but I don't see anyway of doing it. 

What we can do here is to choose symbols which don't occupy more space than their widths. This works for me, 
```elisp 
(setq bibtex-actions-symbols
  `((file  .  ("📄" . "  "))
    (note .   ("✎" . " "))
    (link .   ("🔗" . "  ")))))
```

Previously there was a `tab` character in the affixation which was somehow taking care of the alignment though I don't know how but that causes problems with determining how many columns are available for the rest of the string.

I will make a pr to adjust change the file symbol to adjust this, which also fixes a quoting issue.

The change to file  symbol is not a proper fix since it is likely to be font dependent but this is the usual problem of mixing variable width fonts with character grid based displays.

**Comment by bdarcus** [comment]
Fixed-ish in #249
